src/commands/ext-configure.ts

```diff
@@ -55,16 +115,13 @@ export default new Command("ext:configure <extensionInstanceId>")
       }
       const paramSpecWithNewDefaults =
         paramHelper.getParamsWithCurrentValuesAsDefaults(existingInstance);
-      const immutableParams = _.remove(paramSpecWithNewDefaults, (param) => {
-        return param.immutable || param.param === "LOCATION";
-        // TODO: Stop special casing "LOCATION" once all official extensions make it immutable
-      });
+      const immutableParams = _.remove(paramSpecWithNewDefaults, (param) => param.immutable);
```

Comment:
Consider using partition from functional.ts here instead (https://github.com/firebase/firebase-tools/blob/master/src/functional.ts#L99)

Reply:
good suggestion, wasn't a fan of this lodash function too.. 

https://media.makeameme.org/created/reach-for-immutable.jpg